### PR TITLE
Suraj added his json file

### PR DESCRIPTION
**Pull request by suraj-001**
## Proposed Changes

  -Added my json file .

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/259"></a>

### CODE REVIEW

File: public/data/SurajkAGarwal.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Suraj Kumar Agarwal",
+    "bio": "Coder | PG Student | YT Conten creator|Student of (MCA)NIT Bhopal INDIA,",
+    "avatar": "https://github.com/suraj-001.png",
+    "links": [
+        {
+            "name": "Follow me on GitHub",
+            "url": "https://github.com/suraj-001/LinkFree",
```
Comment (schmelto) [inline public/data/SurajkAGarwal.json:8]:
```suggestion
            "url": "https://github.com/suraj-001",
```

**Review by schmelto — CHANGES_REQUESTED** [review]

File: public/data/SurajkAGarwal.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            "icon": "github"
+        },
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/AgarwalSurajk",
+            "icon": "twitter"
+        },
+        {
+            "name": "Subscribe to my youtube channel",
```
Comment (Panquesito7) [inline public/data/SurajkAGarwal.json:17]:
```suggestion
            "name": "Subscribe to my YouTube channel",
```

File: public/data/SurajkAGarwal.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            "url": "https://www.youtube.com/channel/UC-jJxlbde8o9U2qrpaPOPJw",
+            "icon": "youtube"
+        }
+    ]
+}
```
Comment (Panquesito7) [inline public/data/SurajkAGarwal.json:22]: Missing an endline.
```suggestion
}

```

File: public/data/SurajkAGarwal.json (hunk 1 above)
Comment (Panquesito7) [inline public/data/SurajkAGarwal.json:8]:
```suggestion
            "url": "https://github.com/suraj-001",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by rohan-kulkarni-25 — COMMENTED** [review]
Please don't commit Package-lock.json

**Comment by Panquesito7** [comment]
> Please don't commit Package-lock.json

That's something automatically done by `npm`.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/EddieHubCommunity/LinkFree/pull/259/conflicts).

File: public/data/SurajkAGarwal.json (hunk 2 above)
Comment (suraj-001) [reply public/data/SurajkAGarwal.json:17]: ok sure

File: public/data/SurajkAGarwal.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Suraj Kumar Agarwal",
+    "bio": "Coder | PG Student | YT Conten creator|Student of (MCA)NIT Bhopal INDIA,",
```
Comment (Panquesito7) [inline public/data/SurajkAGarwal.json:3]:
```suggestion
    "bio": "Coder | PG Student | YT Conten creator | Student of (MCA)NIT Bhopal INDIA",
```

File: public/data/SurajkAGarwal.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Suraj Kumar Agarwal",
+    "bio": "Coder | PG Student | YT Conten creator | Student of (MCA)NIT Bhopal INDIA",
```
Comment (Panquesito7) [inline public/data/SurajkAGarwal.json:3]:
```suggestion
    "bio": "Coder | PG Student | YT Content creator | Student of (MCA)NIT Bhopal INDIA",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by rohan-kulkarni-25 — CHANGES_REQUESTED** [review]
- Please don't commit  Package lock json

File: public/data/SurajkAGarwal.json (hunk 5 above)
Comment (suraj-001) [reply public/data/SurajkAGarwal.json]: updated please check

**Comment by Panquesito7** [comment]
> * Please don't commit  Package lock json

That's something done by `npm` automatically.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Comment by suraj-001** [comment]
Hey  after merge my profilr is not reflecting in site there  i tried 
http://linkfree.eddiehub.org/SurajkAGarwal not working plz check and let me know here or on discord server suraj-001

**Comment by Panquesito7** [comment]
> Hey after merge my profilr is not reflecting in site there i tried
> http://linkfree.eddiehub.org/SurajkAGarwal not working plz check and let me know here or on discord server suraj-001

I can see it. For me, there are no problems. 🤔

**Pull request merged**